spec/graphql/mutations/stories/update_story_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+module Mutations
+  module Stories
+    RSpec.describe UpdateStory, type: :request do
+      def query
+        <<~GQL
+        mutation {
+          updateStory(
+            input: {
+              id: #{Story.last.id},
+              title: "New Title",
+              bodyText: "New Text",
+              image: "Sample image",
+              word: "Update",
+              sound: "Alert!",
+              totalTimeInSeconds: 5000
+              })
+              { story {
+                id
+                title
+                bodyText
+                image
+                word
+                sound
+                totalTimeInSeconds
+              }
+            }
+          }
+        GQL
+      end
+
+      def bad_id_query
+        <<~GQL
+        mutation {
+          updateStory(
+            input: {
+              id: 150150
+              title: "New Title",
+              bodyText: "New Text",
+              image: "Sample image",
+              word: "Update",
+              sound: "Alert!",
+              totalTimeInSeconds: 5000
+              })
+              { story {
+                id
+                title
+                bodyText
+                image
+                word
+                sound
+                totalTimeInSeconds
+              }
+            }
+          }
+        GQL
+      end
+
+      def bad_attr_query
+        <<~GQL
+        mutation {
+          updateStory(
+            input: {
+              id: #{Story.last.id}
+              title: "New Title",
+              bodyText: "New Text",
+              image: "Sample image",
+              word: "Update",
+              sound: 10,
+              totalTimeInSeconds: 5000
+              })
+              { story {
+                id
+                title
+                bodyText
+                username
+                image
+                word
+              }
+            }
+          }
+        GQL
+      end
+
+      it 'updates a story' do
+        @story = create(:story)
+
```

Comment:
I don't think this needs to be a global variable, and not sure these factories need to be set to a variable in most cases since we don't call them.